po/ru.po

```diff
@@ -5759,6 +5797,8 @@ msgid ""
 "See [pattern matching](../pattern-matching.md) for more details on patterns in\n"
 "Rust."
 msgstr ""
+"Больше деталей про использование образцов в языке Rust можно найти на\n"
```

Comment:
```suggestion
"Больше информации об использование шаблонов в языке Rust можно найти на\n"
```

Reply:
We should probably add "шаблон" into [the terminology sheet](https://docs.google.com/spreadsheets/d/1T9l_Gd4O7qe3rA5EnK4Rk-7pRO4TutukCuJdakUHDss/edit#gid=168787290), since the term isn't mentioned anywhere in the [rust-lang-ru dictionary](https://github.com/rust-lang-ru/dictionary#readme).